### PR TITLE
(DEV-4204) Adds Extra Line to Notes

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Screen Shot 2019-11-04 at 2 31 10 PM

### What does this PR do, or why is it needed?

This adds an extra carriage return between the header and meat of the exported notes.

*NOTE:* This will probably add an extra, extra line for sermons with communicators, that is due to another issue and will be resolved shortly.

### How do I test this PR?

Export the "Third Option" sermon notes.

## TODO

- [x] I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))
- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by richarddubay** [comment]
@redreceipt This might sound picky, but the problem only existed for guest communicators. It was fine for communicators. So now there is an extra space between the communicator and the notes.
Screenshot 2019-11-04 15 48 53
Screenshot 2019-11-04 15 52 25

**Comment by redreceipt** [comment]
Mark this one blocked until the other is merged in, I think it will fix itself

—
Michael

**Comment by richarddubay** [comment]
Okay smart guy! Nice call!
Screenshot 2019-11-04 16 12 46

**Review by richarddubay — APPROVED** [review]

**Pull request merged**
